.github/workflows/push.yml

```diff
@@ -14,6 +14,10 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      # Install pnpm
+      - name: Install pnpm

```

Comment:
1. I discovered pnpm has its own github action for setup. Please use that instead
2. Look at pull-request.yml
    - `pnpm/action-setup@v4`